### DISCUSSION TITLE
Where is there wallet ? Cüzdanlar nerede?

### DISCUSSION THREAD

**Discussion by Yenal4526** [IBAX-io/go-ibax#622] [Bug Submission]
Friends, as you know, we are connected to the Ibax Network site.  We set up our wallets.  The site is improving itself.  But the Wallets we set up are not showing up.

Arkadaşlar bildiğiniz üzere Ibax Network sitesine bağlıyız.  Cüzdanlarımızı kurduk.  Site kendini geliştiriyor.  Ancak kurduğumuz Cüzdanlar görünmüyor.

**Comment by ibaxio** [comment]
Please describe in detail and include screenshots, thanks.
